pkg/v1/providers/yttcb/clusterbootstrap.yaml

```diff
@@ -0,0 +1,223 @@
+metadata:
+  name: #@ data.values.CLUSTER_NAME
+  namespace: #@ data.values.NAMESPACE
+spec:
+  antrea:
+    config:
+      #@ if data.values.NSXT_POD_ROUTING_ENABLED:
+      trafficEncapMode: "noEncap"
+      noSNAT: true
+      #@ else:
+      trafficEncapMode: #@ data.values.ANTREA_TRAFFIC_ENCAP_MODE
+      noSNAT: #@ data.values.ANTREA_NO_SNAT
+      #@ end
+      disableUdpTunnelOffload: #@ data.values.ANTREA_DISABLE_UDP_TUNNEL_OFFLOAD
+      featureGates:
+        #@ if data.values.NSXT_POD_ROUTING_ENABLED:
+        AntreaProxy: true
+        #@ else:
+        AntreaProxy: #@ data.values.ANTREA_PROXY
+        #@ end
+        EndpointSlice: #@ data.values.ANTREA_ENDPOINTSLICE
+        AntreaPolicy: #@ data.values.ANTREA_POLICY
+        NodePortLocal: #@ data.values.ANTREA_NODEPORTLOCAL
+        AntreaTraceflow: #@ data.values.ANTREA_TRACEFLOW
+        Egress: #@ data.values.ANTREA_EGRESS
+        FlowExporter: #@ data.values.ANTREA_FLOWEXPORTER
+#@ end
+---
+#@ if data.values.CNI == "calico":
+apiVersion: cni.tanzu.vmware.com/v1alpha1
+kind: CalicoConfig
+metadata:
+  name: #@ data.values.CLUSTER_NAME
+  namespace: #@ data.values.NAMESPACE
+spec:
+  calico:
+    config:
+      vethMTU: 0
+#@ end
+---
+#@ if vspherecpi_configs_exist() and data.values.PROVIDER_TYPE == "vsphere" and not data.values.IS_WINDOWS_WORKLOAD_CLUSTER:
+apiVersion: v1
+kind: Secret
+metadata:
+  name: #@ data.values.CLUSTER_NAME + "-vsphere-credential"
+  namespace: #@ data.values.NAMESPACE
+data:
+  password: #@ data.values.VSPHERE_USERNAME
+  username: #@ data.values.VSPHERE_PASSWORD
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: #@ data.values.CLUSTER_NAME + "-nsxt-credential"
+  namespace: #@ data.values.NAMESPACE
+data:
+  password: #@ data.values.NSXT_USERNAME
+  username: #@ data.values.NSXT_PASSWORD
+---
+apiVersion: cpi.tanzu.vmware.com/v1alpha1
+kind: VSphereCPIConfig
+metadata:
+  name: #@ data.values.CLUSTER_NAME
+  namespace: #@ data.values.NAMESPACE
+spec:
+  vsphereCPI:
+    mode: vsphereCPI
+    vCenterAPIEndpoint: #@ data.values.VSPHERE_SERVER
+    datacenter: #@ data.values.VSPHERE_DATACENTER
+    vSphereCredentialLocalObjRef:
+      kind: Secret
+      name: #@ data.values.CLUSTER_NAME + "-vsphere-credential"
+    tlsThumbprint: #@ data.values.VSPHERE_TLS_THUMBPRINT
+    region: #@ data.values.VSPHERE_REGION
+    zone: #@ data.values.VSPHERE_ZONE
+    insecure: #@ data.values.VSPHERE_INSECURE
+    ipFamily: #@ data.values.TKG_IP_FAMILY
+    vmNetwork:
+      excludeInternalSubnetCidr: #@ vsphereControlPlaneEndpointAsCidr
+      excludeExternalSubnetCidr: #@ vsphereControlPlaneEndpointAsCidr
+    tlsCipherSuites: TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+    nsxt:
+      podRoutingEnabled: #@ data.values.NSXT_POD_ROUTING_ENABLED
+      route:
```

Comment:
https://github.com/vmware-tanzu/tanzu-framework/blob/c7fe26b68864c31af558b35e40ada500cac92755/pkg/v1/providers/ytt/02_addons/cpi/cpi_addon_data.lib.yaml#L41-L44
Compared to previous, clusterCIDR is not here, and it will be inferred inside controller？

Reply:
It's not here since it will be derived from the cluster

Reply:
Cool